app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>MiniShopGh</title>
+    <title>Football MiniShop</title>
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
+    <ul class="navbar">
+      <li><a id="site_title">Football MiniShop</a></li>
+      <li><a class="navbar_link" href="/merchants">Merchants</a></li>
+      <li><a class="navbar_link" href="/items">Items</a></li>
+    </ul>
```

Comment:
There's also a nav tag that you could wrap this in as well. I like this idea of putting these links in the application.html.erb so that it persists on all pages within this app.